### PR TITLE
feat($stylesheet): Improved make-row mixin & applied container/row mixins to semantic classes

### PR DESCRIPTION
**Pull request by j0ntsang**

### CODE REVIEW

File: src/styles/_section_image_and_text.scss [hunk 1]
```diff
@@ -1,2 +1,8 @@
 // Image & Text
 // ======================================
+
+.text-image {}
```
Comment (derek-findthebrink) [inline src/styles/_section_image_and_text.scss:4]: unused!

**Review by derek-findthebrink — APPROVED** [review]
YASSSS absolutely delightful!!! Nice dude!!

**Comment by derek-findthebrink** [comment]
You should show this sass-jitsu to @saviogl

**Review by mjjcha — APPROVED** [review]
@j0ntsang - Learning so much about the power of scss through this PR. As far as I can tell, it looks clean! :cool:!

File: src/styles/_section_image_and_text.scss (hunk 1 above)
Comment (j0ntsang) [reply src/styles/_section_image_and_text.scss:4]: There are a bunch of those, they'll get their styles soon ;)

**Pull request merged**